Cargo.toml

```diff
@@ -2,4 +2,5 @@
 members = [
     "openraft",
     "memstore",
+    "example-raft-kv",
```

Comment:
Would you like to instead exclude it? Like this:

```toml
[workspace]
members = [
    "openraft",
    "memstore",
]
exclude = [
    "example-raft-kv"
]
```

Reply:
Looks like a great idea!